src/RevokePhoneNumber.tsx

```diff
@@ -58,7 +58,13 @@ export const RevokePhoneNumber = ({ forwardedRef }: Props) => {
 
   const handleRevokePhoneNumber = async () => {
     ValoraAnalytics.track(SettingsEvents.settings_revoke_phone_number_confirm)
-    await revokeNumberAsync.execute()
+    try {
+      await revokeNumberAsync.execute()
+    } catch (error) {
+      // the error handler for revokeNumberAsync handles the error but it is
```

Comment:
nit: Should we at least have a log message here?
Just in case we are ignoring an unexpected error?

Reply:
i can add a logger error here, but i'm not sure it has real user impact since the `onError` block is triggered. the error just keeps propagating up but i don't think there's much we can do about it